js/main.js

```diff
@@ -317,26 +327,29 @@ function cityIdToLabel(s) {
 /*
  * Set the current city.
  *
- * `city` is the city ID.
+ * `cityID` is the new city's ID. If `newHistoryState` is true then a new
+ * entry in the browser's history is created for the change. Otherwise the
+ * current history entry is replaced.
  */
-function setCity(city) {
-    var filename = 'cities/' + city + '.json';
+function setCity(cityID, newHistoryState) {
```

Comment:
Let's leave it this way. My point was only about: "One could not guess what the boolean in `setCity(getHashCity(), false);` does without looking at the function declaration or even at the implementation."